doc/user_guide/index.rst

```diff
@@ -38,32 +33,85 @@
    -----------------------------------------------------------------------------
    Written by R. W. Ford and A. R. Porter, STFC Daresbury Lab
 
+.. only:: html
+
+    .. image:: psyclone_v1.0.png
```

Comment:
Do we have to make a copy of this image file or is it possible to use the one in `../logo`?

Reply:
Ups... you are right, updated reference and deleted copy